src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/ciagmigration/CiagInductionMigrationService.kt

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.ciagmigration
+
+import mu.KotlinLogging
+import org.springframework.stereotype.Service
+import org.springframework.web.reactive.function.client.WebClient
+
+private val log = KotlinLogging.logger {}
+
+@Service
+class CiagInductionMigrationService(private val ciagWebClient: WebClient) {
+
+  fun migrateCiagInductions() {
```

Comment:
Service method that I'm envisaging will do most of the migration work (or at least the orchestration)
At the moment it simply calls the CIAG API to get a specific prisoner induction (that exists in `dev`)
My thinking is that once we merge this to dev, and then enable the feature toggle, it should call the CIAG API which will prove the `WebClient` including it's authorization header. It will just dump out the ciag induction to the log on startup 👍 

The other thing that might need some thought .... at the moment the response from the API call is a a `Map`. This is OK for now and for the purpose of this here and now, but once we get into the migration code we will probably want a better type .... which might mean we need to import the CIAG API swagger spec in order to codegen their response class type .... 🤔